documentation/Using-Azure-TestFramework.md

```diff
@@ -10,10 +10,11 @@
 4. Record/Playback tests
 5. Change Test Environment settings at run-time
 6. Troubleshooting
-7. Environment Variable Reference
+7. Supported Key=Value pairs in ConnectionString
+8. Environment Variable Reference
 
 ## 1. Getting Started
-1. Launch .\tools\PS-VSPrompt shortcut
+1. Double click .\tools\PS-VSPrompt shortcut
```

Comment:
@shahabhijeet I was thinking this could be

"Double click the `PS-VSPrompt` shortcut located in the **tools** folder of the repository"